### ISSUE TITLE
provide drive letter

### ISSUE THREAD

**Issue by sdolenc** [MicrosoftDocs/OfficeDocs-DeployOffice#78]
Is there a way I can provide the letter of the drive I want to install office to?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f907a7f6-5351-3044-e258-9d18489a08e9
* Version Independent ID: 27d220cf-42e8-1cfc-08aa-586686a9f8b2
* Content: [Configuration options for the Office Deployment Tool](https://docs.microsoft.com/en-us/DeployOffice/configuration-options-for-the-office-2016-deployment-tool)
* Content Source: [DeployOffice/configuration-options-for-the-office-2016-deployment-tool.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/configuration-options-for-the-office-2016-deployment-tool.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @jwhit-MSFT
* Microsoft Alias: **jwhit**

**Comment by sdolenc** [comment]
makes sense. is there a way to request that feature for a future release?

**Comment by sdolenc** [comment]
FWIW: my workaround was
- before installing
- launch disk management
- mount 8gb partition on a separate drive to "C:\Program Files\Microsoft Office"

![workaround](https://user-images.githubusercontent.com/21182133/43666068-2a09d8fa-9727-11e8-9d0b-a90dc013b063.png)

Feel free to resolve/close this issue at your leisure

**Comment by DHB-MSFT** [comment]
That feature request has been made previously but there are no plans to allow installation to other drives. The reasoning behind only allowing installs to the system drive is that the system drive is the only drive guaranteed to remain consistent. Non-system drive letters can change as disks are added or removed, which may prevent Office from working.

**Comment by AphexGwyn** [comment]
Sorry to jump in on this thread but ids it possible to modify the configuration.xml to install Office to a specific folder on the C drive other than the default folder?

**Comment by DHB-MSFT** [comment]
No, you can't specify a different folder for the installation. Thanks for your question.
